### PR TITLE
Add example of setting the random seed.

### PR DESCRIPTION
**Pull request by kresan**
An example of how to set a random seed for both Geant3 and Geant4.
Implemented in examples/simulation/Tutorial1

---

Checklist:

* [x] Rebased against `dev` branch
* [x] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [x] Followed [the seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

File: examples/simulation/Tutorial1/macros/run_tutorial1.C [hunk 1]
```diff
@@ -98,7 +98,13 @@ void run_tutorial1(Int_t nEvents = 10, TString mcEngine = "TGeant3", Bool_t isMT
   // ------------------------------------------------------------------------
 
   // -----   Initialize simulation run   ------------------------------------
+  UInt_t randomSeed = 123456;
+  TRandom3 random(randomSeed);
+  gRandom = &random;
+
   run->Init();
+
+  TVirtualMC::GetMC()->SetRandom(&random);
```
Comment (fuhlig1) [inline examples/simulation/Tutorial1/macros/run_tutorial1.C]: Isn't it enough to use

gRandom->SetSeed(123456) ?

Comment (kresan) [reply examples/simulation/Tutorial1/macros/run_tutorial1.C]: The method TVirtualMC::SetRandom stores the pointer in both gRandom and fRandom variables. If the second one is also used by the transport engine, then it is not enough.

**Comment by fuhlig1** [comment]
That is true but in the constructor of TVirtualMC you have

> fRandom = gRandom;

which sets fRandom to the already existing gRandom. So doing all the changes to gRandom before calling Init() should be enough to properly set everything.

**Comment by kresan** [comment]
@fuhlig1 it is changed now to modifying the gRandom before Init()

**Pull request merged**